diff --git a/subprojects/gst-devtools/debug-viewer/GstDebugViewer/GUI/window.py b/subprojects/gst-devtools/debug-viewer/GstDebugViewer/GUI/window.py
index 43ed9c0810..43d2dc2fc4 100644
--- a/subprojects/gst-devtools/debug-viewer/GstDebugViewer/GUI/window.py
+++ b/subprojects/gst-devtools/debug-viewer/GstDebugViewer/GUI/window.py
@@ -569,10 +569,8 @@ class Window (object):
     @action
     def handle_open_file_action_activate(self, action):
 
-        dialog = Gtk.FileChooserDialog(None, self.gtk_window,
-                                       Gtk.FileChooserAction.OPEN,
-                                       (Gtk.STOCK_CANCEL, Gtk.ResponseType.CANCEL,
-                                        Gtk.STOCK_OPEN, Gtk.ResponseType.ACCEPT,))
+        dialog = Gtk.FileChooserNative.new(None, self.gtk_window,
+                                       Gtk.FileChooserAction.OPEN, None, None)
         response = dialog.run()
         dialog.hide()
         if response == Gtk.ResponseType.ACCEPT:
